src/props/formats/html.jsx

```diff
@@ -229,50 +303,63 @@ let Styleguide = React.createClass({
+    );
   },
 
   render(json) {
     return (
-      <html>
+      <html lang="en">
         <head>
           <meta charSet="utf-8" />
           <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
           <title>Design Tokens</title>
           <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.2.0/normalize.min.css" />
           <Style />
         </head>
         <body>
           <header role="banner" className="banner">
             <div className="container">
-              <h1>Design Attribute Guide</h1>
+              <h1>Design Tokens</h1>
             </div>
           </header>
+
           <div className="container">
             <main role="main">
-              <table>
-                <tbody>
-                  {this.renderSection('text-color', 'Text Colors')}
-                  {this.renderSection('background-color', 'Background Colors')}
-                  {this.renderSection('gradient', 'Gradients')}
-                  {this.renderSection('border-color', 'Border Colors')}
-                  {this.renderSection('border-style', 'Border Styles')}
-                  {this.renderSection('hr-color', 'Horizontal Rule Colors')}
-                  {this.renderSection('drop-shadow', 'Drop Shadows')}
-                  {this.renderSection('inner-shadow', 'Inner Drop Shadows', this.renderDropShadow)}
-                  {this.renderSection('font', 'Fonts')}
-                  {this.renderSection('font-weight', 'Font Weights')}
-                  {this.renderSection('font-size', 'Font Sizes')}
-                  {this.renderSection('line-height', 'Line Heights')}
-                  {this.renderSection('spacing', 'Spacing')}
-                  {this.renderSection('radius', 'Radius')}
-                  {this.renderSection('time', 'Time')}
-                  {this.renderSection('media-query', 'Media Queries')}
-                </tbody>
-              </table>
+              {this.renderSection('sizing', 'Sizing')}
+              {this.renderSection('spacing', 'Spacing')}
+              {this.renderSection('font', 'Fonts')}
+              {this.renderSection('font-style', 'Font Styles')}
+              {this.renderSection('font-weight', 'Font Weights')}
+              {this.renderSection('font-size', 'Font Sizes')}
+              {this.renderSection('line-height', 'Line Heights')}
+              {this.renderSection('font-families', 'Font Families')}
+              {this.renderSection('border-style', 'Border Styles')}
+              {this.renderSection('border-color', 'Border Colors')}
+              {this.renderSection('radius', 'Radius')}
+              {this.renderSection('border-radius', 'Border Radii')}
+              {this.renderSection('hr-color', 'Horizontal Rule Colors')}
+              {this.renderSection('background-color', 'Background Colors')}
+              {this.renderSection('gradient', 'Gradients')}
```

Comment:
@kaelig is there a way to deprecate a category? I wanted to remove the categories that we don't use anymore, but not sure if this will break things for people.